src/components/Ticker/Ticker.js

```diff
@@ -144,23 +143,39 @@ const Ticker = ({ images, arrowLeft, arrowRight }) => {
   const stop = () => {
     isRunning.current = false;
   };
-
   return (
     <div ref={container} className={styles.container}>
       <a.ul
         ref={ticker}
         style={{ transform: props.x.interpolate(iTranslate) }}
         className={styles.ticker}
       >
-        {items.map(({ childImageSharp }, index) => {
-          return (
-            <li key={index} className={styles.item}>
-              <div className={styles.card}>
-                <Img fluid={childImageSharp.fluid} draggable={false} />
-              </div>
-            </li>
-          );
-        })}
+        {type === 'works'
+          ? items.map((item, index) => {
+              return (
+                <li className={styles.itemWorks} key={item + index}>
```

Comment:
`item + index` returns strings like `[object Object]1`

I think the best way here would be to add custom names to the object with photos, e.g.

```
let works = {
    web: {
        name: 'web',
        element: <div> <image /> </div>
    },
    investmentsTravelSomethingElse: {
        name: 'investmentsTravelSomethingElse',
        element: <div> <image /> <image /> <image /> </div>
    }
}

```